lint/base_linter/node_linter.py

```diff
@@ -47,6 +47,22 @@ def smart_paths_upwards(start_dir):
     )
 
 
+def is_yarn_project(path, manifest):
+    # type: (str, Dict[str, Any]) -> bool
+    package_manager = manifest.get('packageManager')
+    if isinstance(package_manager, str):
+        if package_manager.startswith('@'):
+            package_manager = package_manager[1:]
+        name = package_manager.split('@', 1)[0]
```

Comment:
So the expected string is `@yarn@3.5` or `@yarn` etc.?  Make sure to add a test for it.

Reply:
Done.